docs/contributing/Compiler Test Plan.md

```diff
@@ -308,6 +308,8 @@ __makeref( x )
 - Interpolated string
 - Tuple literal
 - Tuple
+- Default literal
+- Implicit object creation
```

Comment:
Consider putting `(target-typed new)` in here as well, as that's what we've been calling it.

Reply:
The reason I used that name is because the spec currently says "there is a new *object creation conversion* that is an implicit conversion from expression"
https://github.com/dotnet/csharplang/blob/master/proposals/target-typed-new.md

Reply:
Sure. But we call it target-typed new. I'm not saying remove the name, I'm just saying add a thing in parenthesis for ease of future reading.

Reply:
Okay that works